apps/meteor/ee/app/livechat-enterprise/server/lib/AutoTransferChatScheduler.ts

```diff
@@ -25,30 +31,52 @@ class AutoTransferChatSchedulerClass {
 		this.scheduler = new Agenda({
 			mongo: (MongoInternals.defaultRemoteCollectionDriver().mongo as any).client.db(),
 			db: { collection: SCHEDULER_NAME },
+			defaultLockLifetime: 1000, // 1 minute
```

Comment:
[not for this line]
Why this class doesn't extend from the abstract one? :eyes: